src/runtime/contrib/tensorrt/tensorrt_ops.cc

```diff
@@ -20,6 +20,8 @@
  * \file runtime/contrib/tensorrt/tensorrt_ops.cc
  * \brief Converters from Relay ops into TensorRT layers. Converters should
  * inherit from TensorRTOpConverter and implement the Convert() method.
+ * Minimal version 7, 0, 0
+ *  TRT_VERSION_GE(5, 1, 5)
```

Comment:
Please update the TRT document accordingly.
(https://tvm.apache.org/docs/how_to/deploy/tensorrt.html)